tests/test_RestartSerialization.cpp

```diff
@@ -440,14 +442,17 @@ namespace {
 
 struct AquiferFixture {
     AquiferFixture() {
-        using TT = Opm::Properties::TTag::TestRestartTypeTag;
+        using namespace Opm;
+        using TT = Properties::TTag::TestRestartTypeTag;
         const char* argv[] = {
             "test_RestartSerialization",
             "--ecl-deck-file-name=GLIFT1.DATA"
         };
-        Opm::AdaptiveTimeStepping<TT>::registerParameters();
-        Opm::setupParameters_<TT>(2, argv, /*registerParams=*/true);
-        Opm::EclGenericVanguard::setCommunication(std::make_unique<Opm::Parallel::Communication>());
+        AdaptiveTimeStepping<TT>::registerParameters();
+        BlackoilModelParameters<TT>::registerParameters();
```

Comment:
why the following two extra lines?
```c++
        BlackoilModelParameters<TT>::registerParameters();
        EWOMS_REGISTER_PARAM(TT, bool, EnableTerminalOutput, "Do *NOT* use!");
```

Reply:
Is it also because the new "TestTypeTag.hh"?

Reply:
They used to be done in the EbosProblem class but since that is no longer a thing, I have to do them here instead.